kubespawner/spawner.py

```diff
@@ -274,6 +291,32 @@ def _hub_connect_port_default(self):
         """
     )
 
+    storage = Unicode(
```

Comment:
Call it user_storage_capacity for clarity?

Also let's default this to None (and set `allow_none=True` to make that possible), and not do anything if it is None?